pkg/plugins/platform/utils.go

```diff
@@ -128,32 +128,6 @@ func ComposeFileExists(app App) bool {
 	return true
 }
 
-// Cleanup will clean up ddev apps even if the composer file has been deleted.
-func Cleanup(app App) error {
```

Comment:
It looks like this has changed over time, and was never updated. Previously we were relying on container names (yuck) instead of labels. It feels like it would be fairly easy to re-write this to use labels + the docker API for stopping, and would be very valuable to have in the case of a missing or corrupted docker-compose file. I'd be happy to take on the re-write if you don't have the bandwidth